db/data/20211014090307_add_proceeding_types.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddProceedingTypes < ActiveRecord::Migration[6.1]
+  def up
+    ProceedingType.populate
+    puts 'populated proceeding types'
+  end
+
+  def down
+    ProceedingType.destroy_all
+    ScopeLimitation.destroy_all
```

Comment:
We're destroying ScopeLimitations but not populating them in the Up section, are they seeded from within the ProceedType and, if so, should we destroy them via a rails belongs_to/has_many link?